src/compiler/crystal/syntax/lexer.cr

```diff
@@ -493,6 +493,8 @@ module Crystal
             case char
             when '\\'
               case char = next_char
+              when 'a'
+                io << "\u{7}"
```

Comment:
Is there a reason these were not touched by #5882 @sija?

Reply:
Nope, just an omission.

Reply:
Should I make a PR to also make these strings chars?

Reply:
I'd suggest to merge this one first.